### PR TITLE
Fixing values for helm

### PR DESCRIPTION
**Pull request by amitschendel**

### CODE REVIEW

File: charts/kubeenforcer/values.yaml [hunk 1]
```diff
@@ -38,10 +36,9 @@ podAnnotations: {}
 podSecurityContext: {}
 
 securityContext:
+  runAsUser: 0
   allowPrivilegeEscalation: false
   readOnlyRootFilesystem: true
-  capabilities:
-    drop: ["ALL"]
```
Comment (slashben) [inline charts/kubeenforcer/values.yaml:44]: Leave `drop ["ALL"]` and add `add ["NET_ADMIN"]`

File: charts/kubeenforcer/values.yaml [hunk 2]
```diff
@@ -38,10 +36,9 @@ podAnnotations: {}
 podSecurityContext: {}
 
 securityContext:
+  runAsUser: 0
```
Comment (slashben) [inline charts/kubeenforcer/values.yaml]: No need for this line

File: charts/kubeenforcer/values.yaml (hunk 1 above)
Comment (amitschendel) [reply charts/kubeenforcer/values.yaml:44]: done

File: charts/kubeenforcer/values.yaml (hunk 2 above)
Comment (amitschendel) [reply charts/kubeenforcer/values.yaml]: done

**Review by slashben — APPROVED** [review]

**Pull request merged**